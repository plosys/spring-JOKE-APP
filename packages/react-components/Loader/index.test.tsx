
import { render } from '@testing-library/react';

import { Loader } from './index.js';

describe('Loader', () => {
  it('should match its snapshot', () => {
    const { container } = render(<Loader />);
    expect(container).toMatchSnapshot();
  });
});