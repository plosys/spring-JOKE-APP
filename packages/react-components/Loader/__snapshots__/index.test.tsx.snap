
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match its snapshot 1`] = `
<div>
  <div
    class="loader appsemble-loader"
  />
</div>
`;